### PR TITLE
Add message when user doesn't have a browser with websockets

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/internal-wip/issues/22.

As previously discussed, as a temporary solution, we just inform the user that they need to use websockets when doing the research experiment. I think we should change the copy on this modal to be better, but can't think of anything right now.

## Screenshots
Screen Shot 2020-02-26 at 4 41 34 PM

### CODE REVIEW

File: app/javascript/src/experiment-solution.js [hunk 1]
```diff
@@ -26,6 +28,13 @@ class ExperimentSolution {
     this._setupSplit();
   }
 
+  _checkWebsocketsCompatibility() {
+    if(!('WebSocket' in window) && !('MozWebSocket' in window)) {
```
Comment (kntsoriano) [inline app/javascript/src/experiment-solution.js]: I got this snippet from https://stackoverflow.com/questions/20060320/what-is-the-best-way-to-detect-websocket-support-using-javascript.

**Comment by iHiD** [comment]
@ErikSchierboom Do things still work ok in the latest MS browser?

**Comment by ErikSchierboom** [comment]
Yes, I've tested on IE 11 and that worked fine. I didn't see a popup :)

**Comment by iHiD** [comment]
Great. Thanks :)

**Pull request merged**